kss-builder/index.hbs

```diff
@@ -212,14 +212,12 @@
 <footer role="contentinfo">
   <div class="wrapper">
     <div class="footer-logo">
-      <img src="https://www.gov.au/alpha/assets/coatofarms-01.svg" alt="Australian Government Coat of Arms" />
+      <img alt="Australian Government Coat of Arms" src="../assets/img/logos/coatofarms-01.svg">
```

Comment:
See gov.au/alpha for "onerror="this.src='/alpha/assets/coatofarms-01.png'; this.onerror=null;" workaround for IE8.

It might be better to just inline this logo as well otherwise we begin to need doco about downloading the image assets, where to put them relative to CSS etc.